spec/features/user_creates_a_dinosaur_spec.rb

```diff
@@ -0,0 +1,107 @@
+require 'rails_helper'
+
+feature 'user can create a new dinosaur', %{
+  As a user
+  I want to create a new dinosaur
+  So that I can share my favorite dinosaurs
+
+  Acceptance Criteria
+  [√] User can navigate to new dinosaur form from index
+  [√] User can view form properly
+  [√] User gets confirmation if dinosaur is entered correctly
+  [√] User gets errors if form is filled out incorrectly
+  [√] User gets redirected to show page of created dinosaur
+
+} do
+
+  feature "User is not signed in" do
+
+    scenario "User is asked to sign in before creating a dinosaur" do
+      visit new_dinosaur_path
+      expect(page).to have_content("Sign Up Sign In")
+    end
+
+  end
+
+  feature "User is signed in" do
+
+    before(:each) do
+      user = FactoryGirl.create(:user)
+      visit new_user_session_path
+      fill_in 'Email', with: user.email
+      fill_in 'Password', with: user.password
+      click_button 'Log in'
+    end
+
+    scenario "User can navigate to form from index" do
+      visit root_path
+      click_link "Create a dinosaur"
+      expect(page).to have_content("Create a Dinosaur!")
+
+    end
+
+    scenario "Form fields are visible" do
+
+      visit new_dinosaur_path
+      expect(page).to have_content("Name")
+      expect(page).to have_content("Location found")
+      expect(page).to have_content("Info url")
```

Comment:
I'd use `have_field` here instead of `have_content`. It's just a little more descriptive.